_sass/_csl.scss

```diff
@@ -22,6 +22,10 @@
 
 	margin-bottom: 30px;
 
+	ul {
+		margin-top: 0px;
```

Comment:
Sure, that's fine :-)

Tiny detail: when a property's value is zero the unit is optional, so can write margin-top: 0; and skip two keystrokes ;-)

Reply:
Okay, thanks!